src/core/scene/a-scene.js

```diff
@@ -518,3 +514,19 @@ function exitFullscreen () {
     document.webkitExitFullscreen();
   }
 }
+
+/**
+ * Determines if renderer anti-aliasing should be enabled.
+ * Enabled by default if has native WebVR or is desktop.
```

Comment:
can this be mentioned in the Docs? (especially if you go the `antialias` component route)